_posts/2024-03-01-pyos-newsletter-march-2024.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: single
+title: "pyOpenSci Newsletter: March 2024"
+excerpt: "New partnerships, packages, and conference announcements!"
+author: "Jesse Mostipak"
+permalink: /blog/pyos-newsletter-march-2024.html
+header:
+  overlay_color: "#666"
+  overlay_filter: 0.6
+categories:
+  - blog-post
+  - community
+toc: true
+comments: true
+---
+
+## :star: Welcome, AstroPy
+
+pyOpenSci has officially partnered with the [AstroPy Project](https://www.astropy.org/) through our [Community Partnership program](https://www.pyopensci.org/partners.html). AstroPy is a Community Python Library for Astronomy, and the AstroPy Project is a community effort to develop a single core package for Astronomy in Python, while also fostering interoperability between Python astronomy packages. We're thrilled to be partnering with the AstroPy project, and can't wait to share more!
+
+
+## :purple_heart: Hello, sunPy
+
+[sunPy](https://sunpy.org/), a Python package that provides fundamental tools for accessing, loading, and interacting with solar physics data in Python, has successfully gone through the pyOpenSci open peer review process, which is the first step in our budding partnership with the heliophysics community!
+
+You can read more about sunPy and pyOpenSci in [this lovely blog post](https://sunpy.org/posts/2024/2024-01-24-pyopensci) from Nabil Friej.
+
+## :wave: Say hi to sciform
+
+The [sciform package](https://sciform.readthedocs.io/en/stable/), which is used to convert Python numbers into strings according to a variety of user-selected scientific formatting options, was accepted into the pyOpenSci ecosystem! sciform, like all of the pyOpenSci packages, completed the [pyOpenSci open peer review process](https://www.pyopensci.org/about-peer-review/index.html). And the community support around getting the sciform package review-ready was amazing to see!
+
+Have a package you’ve developed, but aren’t sure it’s in scope for pyOpenSci? Complete a [pre-submission inquiry](https://github.com/pyOpenSci/software-submission/issues/new?assignees=&labels=presubmission&projects=&template=presubmission-inquiry.md&title=) and one of our editors will be in touch!
+
+## :airplane: We're going to [PyCon US](https://us.pycon.org/2024/)
+Will we see you there? Our Executive Director and Founder, [Leah Wasser](https://github.com/lwasser), had her talk about Python packaging for scientists accepted for this May's conference. We've seen a sneak peek of her talk, and guarantee that you won't want to miss it! Be sure to [register today](https://us.pycon.org/2024/accounts/login/?next=/2024/registration/register)!
```

Comment:
Can we add the link to the abstract here: 

https://us.pycon.org/2024/schedule/presentation/34/

also you are too kind 😆 i haven't started my talk yet but plan to 🙃

Reply:
it's going to be amazing!